recipe/build.sh

```diff
@@ -1,2 +1,16 @@
 set -ex
+
+rm -f skimage/filters/*.c
+rm -f skimage/restoration/_*.c
+rm -f skimage/segmentation/*.c
+rm -f skimage/io/*.c
+rm -f skimage/graph/*.c
+rm -f skimage/measure/*.c
+rm -f skimage/_shared/*.c
+rm -f skimage/morphology/*.c
+rm -f skimage/feature/*.c
+rm -f skimage/transform/*.c
+rm -f skimage/draw/*.c
+rm -f skimage/future/*.c
```

Comment:
This works, but I think you should be able to run `make clean` and the Makefile in the source distribution should take care of this.

Reply:
Can we do something like,

```shell
find . -name *.c -print0 | xargs -0 rm -rf
```

instead? Those 14 lines seems like a nightmare to maintain if upstream move those files around.

Reply:
So what we need to do is find all the `.pyx` files and remove associated `.c` files.  But I am not an awk/sed person to make that happen.  (If we remove all the `.c` files then we remove a couple that are not cythonized, and that's bad ... trust me I tried!)
Or maybe it is better to add `make` as a dependency and do `make clean`?

Reply:
> and that's bad ... trust me I tried!

Yep. That can happen if some .c files are not the cythonized ones.

> Or maybe it is better to add make as a dependency and do make clean?

That is always preferred  b/c we outsource the responsibility to upstream ;-p

Reply:
The `make clean` call just runs the following few commands:
`https://github.com/scikit-image/scikit-image/blob/main/Makefile#L9-L11`
where shell script to help remove the cythonized files is here: https://github.com/scikit-image/scikit-image/blob/main/tools/rm_pyx_assoc_c_cpp.sh

Reply:
> find . -name *.c -print0 | xargs -0 rm -rf


The problem there is that there are a couple of `.c` files like `skimage/restoration/unwrap_*.c` that are not generated by Cython and must not be removed. I am trying in #87 with a call to `rm_pyx_assoc_c_cpp.sh` that only removes .c or .cpp files with a corresponding .pyx.